torchbenchmark/models/Super_SloMo/requirements.txt

```diff
@@ -1,3 +1 @@
-torch
```

Comment:
whoops i'll separate these torch/torchvision changes out. it was an accidental part of the commit, i think no longer needed anyway.  (@xuzhao9 did some work on the main installer to help with the issue i was having here)